src/components/step-sequence/step-sequence.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import './step-sequence.scss';
+import './step-sequence-orientation.scss';
+
+const baseClass = 'carbon-step-sequence';
+const classes = orientation => `${baseClass} ${baseClass}--${orientation}`;
+
+const StepSequence = ({ children, orientation = 'horizontal' }) => (
```

Comment:
for consistency you should probably use `defaultProps`? at first I was confused where the default was coming from as I couldn't find `defaultProps` defined.